docs/installation.rst

```diff
@@ -0,0 +1,236 @@
+.. _installing_ansible_navigator:
+
+Installing ansible-navigator with execution environment support
+###############################################################
+
+.. contents::
+   :local:
+
+Linux
+*****
+
+Requirements
+============
+
+* Either ``podman`` or ``Docker for Linux``
+* Internet access (during initial installation)
+
+Install the desired container engine for execution environment support
+======================================================================
+
+* Follow the ``podman`` installation instructions for the appropriate `Linux distribution <https://podman.io/getting-started/installation>`__.
+* Follow the ``Docker for Linux`` installation instructions for the appropriate `Linux distribution <https://docs.docker.com/engine/install/>`__.
+
+Install ansible-navigator
+=============================
+
+#. Install the python package manager using the system package installer (e.g.):
+
+   .. code-block:: console
+
+    sudo dnf install python3-pip
+
+#. Install ansible-navigator:
+
+   .. code-block:: console
+
+    python3 -m pip install ansible-navigator --user
+
+#. Add the installation path to the PATH (e.g.):
+
+   .. code-block:: console
+
+    echo 'export PATH=$HOME/.local/bin:$PATH' >> ~/.zshrc
+
+#. Open a new shell or update the current path (e.g.)
+
+   .. code-block:: console
+
+    source ~/.zshrc
+
+#. Launch ansible-navigator:
+
+   .. code-block:: console
+
+    ansible-navigator
+
+#. ``ansible-navigator`` triggers a one-time download of the demo execution-environment image.
+
+macOS
+*****
+
+Requirements
+============
+
+* Docker desktop for Mac
+* macOS command line developer tools
+* Internet access (during initial installation)
+
+Install the desired container engine for execution environment support
+======================================================================
+
+* Follow the `Docker Desktop for Mac <https://hub.docker.com/editions/community/docker-ce-desktop-mac>`__ installation instructions
+
+.. note::
+
+   There is no convenient way to use ``ansible-navigator`` with ``podman`` on macOS.  Native source mounts from macOS through the ``podman`` machine into the execution environment are not currently available.
+
+   See this `related issue <https://github.com/containers/podman/issues/8016>`__ for details.
+
+Install ansible-navigator
+=========================
+
+#. Install the command line developer tools and proceed with the installation if prompted.
+
+   .. code-block:: console
+
+    xcode-select install
+
+#. Install ansible-navigator:
+
+   .. code-block:: console
+
+    pip3 install ansible-navigator --user
+
+#. Add the installation path to the PATH:
+
+   .. code-block:: console
+
+    echo 'export PATH=$HOME/Library/Python/3.8/bin:$PATH' >> ~/.zshrc
```

Comment:
This time it's probably okay to default to zsh because that's the default macOS shell.